### PR TITLE
More robust argument parsing using minimist

### PR DESCRIPTION
**Pull request by SCdF**
Args can now be passed in any order, using minimist. CLI is backwards compatible with existing conventions, and also allows for using = if people prefer.

e.g.:
   --instance foo.bar
and
   --instance=foo.bar

Are equivalent.

In testing I also found the changelog command was broken so I fixed that while I was in there.

### CODE REVIEW

**Comment by SCdF** [comment]
NB: leveldown fails to compile with node 12. I've dropped support for it in this PR

File: .travis.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ install:
   - "pip install --user git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic"
 before_script:
   - "! grep -Fq '.only(' test/**/*.spec.js"
-  - npm install
+  - npm ci
```
Comment (SCdF) [inline .travis.yml]: So we are actually testing with the versions in the package lock

**Comment by SCdF** [comment]
(things this PR doesn't do that others are welcome to as obvious next steps: pull out the CLI parsing into a unit testable module, use a more complicated CLI framework that integrates and generates help text etc, refactor any of the actual execution code)

File: src/bin/medic-conf.js [hunk 2]
```diff
@@ -14,81 +14,95 @@ const supportedActions = require('../cli/supported-actions');
...
+  return process.exit(0);
+}
+
+if (argv.version) {
+  console.log(require('../../package.json').version);
+  return process.exit(0);
+}
+
+if (argv.changelog) {
+  console.log(fs.read(`${__dirname}/../../CHANGELOG.md`));
```
Comment (SCdF) [inline src/bin/medic-conf.js]: In case the diff doesn't make it clear this got an extra `../` to work properly

**Comment by kennsippell** [comment]
We should AT changes before merge. 

This has caused some regressions. 
https://github.com/medic/medic-conf/issues/185
https://github.com/medic/medic-conf/issues/182

**Pull request merged**